### DISCUSSION TITLE
Very specific cron configration

### DISCUSSION THREAD

**Discussion by lesnerd** [quartznet/quartznet#1484] [Q&A]
I need a cron configuration that will fire; every MON,TUE,SUN from 19:17 to 23:46 every 10 minutes.
I came up with this:
0 17/10 19-23 ? * MON,TUE,SUN *

The one thing I am missing is how to stop it at 23:46. (Now it will stop at the end of 23 which is 23:59)

Please help!!

**Comment by lahma** [comment]
You can augment your schedule with an `ICalendar` instance, it allows you to decide whether suggested schedule is included or not.

**Reply by lesnerd** [reply]
Not sure I understand.
I have no problem including a schedule. What I am missing is how can I specify the ending period of the Quartz even more precisely to the minute.
Is there an example I can take a look at? Whether it is cron or ICalendar?
I am lost 😱

**Reply by lahma** [reply]
You can use calendars to narrow further the firing of a trigger. See [DailyCalendar](https://github.com/quartznet/quartznet/blob/main/src/Quartz/Impl/Calendar/DailyCalendar.cs) which should help here.
